DependencyInjection/HBStampieExtension.php

```diff
@@ -32,12 +32,11 @@ public function load(array $configs, ContainerBuilder $container)
         $processor = new Processor();
         $config    = $processor->processConfiguration(new Configuration($container->getParameter('kernel.debug')), $configs);
 
-        $adapterServiceId = sprintf('hb_stampie.adapter.%s', $config['adapter']);
-        $mailerServiceId  = sprintf("hb_stampie.mailer.%s", $config['mailer']);
 
-        if (!$container->has($adapterServiceId)) {
-            throw new \InvalidArgumentException(sprintf('Invalid adapter "%s" specified', $config['adapter']));
-        }
+        $bundles = $container->getParameter('kernel.bundles');
```

Comment:
Let’s not do this hack. We should only assume that the string the user provided in the config is a proper service of a HTTPlugCliet